packages/platforms/accton/x86-64/wedge100bf-32x/onlp/builds/x86_64_accton_wedge100bf_32x/module/src/platform_lib.c

```diff
@@ -275,31 +286,47 @@ int
 bmc_command_read_int(int* value, char *cmd, int base)
 {
     int len;
-    int i;
+    int i, retry, ret;
     char *prev_str = NULL;
     char *current_str= NULL;
-    if (bmc_send_command(cmd) < 0) {
-        return ONLP_STATUS_E_INTERNAL;
-    }
-    len = (int)strlen(cmd);
-    prev_str = strstr(tty_buf, cmd);
-    if (prev_str == NULL) {
-        return -1;
-    }
-    for (i = 1; i <= TTY_RETRY; i++) {
-        current_str = strstr(prev_str + len, cmd);
-        if(current_str == NULL) {
-            if( !chk_numeric_char(prev_str + len, base) ){
-                return -1;
-            }
-            *value = strtoul(prev_str + len, NULL, base);
-            break;
-        }else {
-            prev_str = current_str;
+
+    ret = -1;
+    retry = TTY_RETRY;
+    do {
+        tty_clear_rxbuf();
+        retry--;
+        if (bmc_send_command(cmd) < 0) {
+            return ONLP_STATUS_E_INTERNAL;
+        }
+
+        len = (int)strlen(cmd);
+        prev_str = strstr(tty_buf, cmd);
+        if (prev_str == NULL) {
+            ret = -1;
             continue;
         }
-    }
-    return 0;
+
+        for (i = 1; i <= TTY_RETRY; i++) {
+            current_str = strstr(prev_str + len, cmd);
+            if(current_str == NULL) {
+                if( !chk_numeric_char(prev_str + len, base) ){
+                    ret = -1;
+                    continue;
+                }
+                *value = strtoul(prev_str + len, NULL, base);
+                ret = 0;
+                goto exit;
+            }else {
+                prev_str = current_str;
+                ret = -1;
+                continue;
+            }
+        }
+        ret = -1;
+    }while(retry);
```

Comment:
Please follow the Linux kernel coding style.

Reply:
Do you have suggested astyle parameter? or other formatter tools?

Reply:
The section [*Pitfalls*](https://www.kernel.org/doc/html/latest/process/4.Coding.html?#pitfalls) references [clang-format](https://www.kernel.org/doc/html/latest/process/clang-format.html).